src/components/layout/ModalContent.tsx

```diff
@@ -66,7 +68,15 @@ export const ScrollModal = ({ children, style, ...props }: ScrollSectionProps) =
   )
 }
 
-const ModalContentStyled = styled.View`
-  padding-top: 10px;
-  padding-bottom: 10px;
-`
+const getDefaultProps = ({ verticalGap, contentContainerStyle }: ModalContentProps) => ({
```

Comment:
Nit: the `contentContainerStyle` properties isn't used in this file (only `verticalGap? is passed to `getDefaultProps`). You wanted to keep it for potential future use?

Reply:
It is being used in line 80!

Reply:
ah sorry, my bad

Reply:
Updated! 😅